time/time_test.go

```diff
@@ -28,7 +28,7 @@ func TestScan(t *testing.T) {
 		{
 			name:     "time.Time",
 			input:    time.Date(2023, 1, 2, 3, 4, 5, 6, time.Local),
-			expected: time.Date(2023, 1, 2, 3, 4, 5, 6, time.UTC),
+			expected: time.Date(2023, 1, 2, 3, 4, 5, 6, time.Local).UTC(),
```

Comment:
This would be a better test (less flakey and easier to read) if it used a fixed time offset instead of `time.Local` and relied on a hardcoded expected value instead of using `.UTC()` function.